### DISCUSSION TITLE
github的记录怎么实现的

### DISCUSSION THREAD

**Discussion by 2010363329** [ant-design/ant-design#46675] [Q&A]
<img width="676" alt="image" src="https://github.com/ant-design/ant-design/assets/76477581/215cc0d3-b971-4e2a-905a-f8e725e4e957">
这个东西没有响应的组件吗 还是说这个功能需要自己去自定义实现

**Comment by tumine** [comment]
您好！我已收到您发送的邮件，稍后我会查看邮件，如果有进一步需求，我会与您再联系。谢谢！

**Comment by RedJue** [comment]
为啥来这里提问？

**Reply by 2010363329** [reply]
我以为ant 有对应的组件 我没找到 所以咨询一下

**Comment by li-jia-nan** [comment]
样式直接复制 GitHub F12，数据来源可以用爬虫爬一下 html 源代码里面的 table => tr => td 标签，我记得有现成的第三方库

**Reply by 2010363329** [reply]
好滴 感谢  我去搜一下

**Reply by 2010363329** [reply]
哪个库有这个 我在ant各种库里都没找到
